test/unit-tests/common/dataset-common/glossary.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+glossary test
+-------------
+
+This section contains references to the :term:`something` and :term:`else`
+glossary entries.
+
+It also contains the multiple references :term:`multiple` and :term:`entries`
+to a ginel glossay entry plus the :term:`nowhere` that is a mistake and
```

Comment:
It'd be nice to cleanup some spelling from this test case (i.e. existant, ginel, glossay).

Reply:
Wow, my fingers were even more stubby and incapable of hitting the correct keys than usual.